Add scene scraper for avjiali.com

bdbenim opened the pull request:
Created a new scene scraper for avjiali.com. Scrapes title, details, performers, tags, code, and image.

JackDawson94 reviewed (commented):
Tested - works.

The website has the same template a the new Baberotica website. Our current Baberotica scraper is broken since they changed the layout.

Can you maybe merge the two (add this to baberotica.yml) ? Not sure if they have migrated all the sites in the baberotica.yml to the new template though...

bdbenim commented:
Ok, I've made a couple of changes that I think fix it:

- Added avjiali.com to baberotica.yml, and copied the xpath selectors over from my original commit
- Split AvIdolz and Teen Thais off into a new scraper avidolz.yml using the original baberotica.yml settings, as these don't seem to have switched to the new layout

I've tested it on a few random scene urls from different studios and I think everything is working as expected. It grabs a studio code for AV Jiali only, which I think should be the intended behaviour as the other studios don't really use codes.

JackDawson94 commented on scrapers/Baberotica.yml:
```diff
@@ -2,42 +2,53 @@ name: Baberotica
           - replace:
               - regex: ^
                 with: "https:"
+      Code:
```
This works for avjiali but not for baberotica

bdbenim replied:
I'm not able to get baberotica codes in the same way. AV Jiali's studio codes happen to be in the URL of the cover image, so I grab them from there, but baberotica doesn't do this. I don't see anywhere else on the page to get it, so I figured it's better to at least have it for avjiali than none at all. All the scenes in stashdb that have codes (from my quick glance) seem to be from the sexlikereal.com domain rather than baberotica.com or baberoticavr.com, so that might have something to do with it.

JackDawson94 commented on scrapers/avidolz.yml:
```diff
@@ -0,0 +1,39 @@
+          - replace:
+              - regex: ^
+                with: "https:"
+      Studio:
```
This is not working, I am receiving the unmapped urls as studio names

Maista6969 commented:
Closing this because of #1481

bdbenim commented:
> Closing this because of #1481

Just wondering if there's something I should do differently for future PRs? This one sat here for 3 months and now is being closed because of a more recent PR, so I'm assuming that there is something wrong what I did.

Maista6969 commented:
Looking at the history of this PR I can't see that you did anything wrong: you were simply the victim of circumstance because the person who had previously been mostly responsible for managing this repository kind of faded out, probably due to other commitments in their real life

I know how frustrating this can be and I'd like to apologize for the delay and subsequent closure of your pull request: I started shepherding this repo a few months ago and have tried to keep on top of things, but the backlog of PRs is one of the things I am still working on resolving

bdbenim commented:
> Looking at the history of this PR I can't see that you did anything wrong: you were simply the victim of circumstance because the person who had previously been mostly responsible for managing this repository kind of faded out, probably due to other commitments in their real life
> 
> I know how frustrating this can be and I'd like to apologize for the delay and subsequent closure of your pull request: I started shepherding this repo a few months ago and have tried to keep on top of things, but the backlog of PRs is one of the things I am still working on resolving

Ah gotcha. No worries! I appreciate you stepping up - it's a lot of work and usually a thankless task, so I have a lot of respect for people who are willing to put in that effort.
